stl/inc/format

```diff
@@ -1266,6 +1270,31 @@ struct _Default_arg_formatter {
     }
 };
 
+// Visitor used for replacement fields that contain specs
+template <class _OutputIt, class _CharT>
+struct _Arg_formatter {
+    using _Context = basic_format_context<_OutputIt, _CharT>;
+
+    _Context* _Ctx                                 = nullptr;
+    _Basic_format_specs<_CharT>* _Specs            = nullptr;
+    basic_format_parse_context<_CharT>* _Parse_ctx = nullptr;

```

Comment:
I did indeed, and went back and forth a few times. I usually don't like to use reference members, but given the number of "is this pointer not null" checks in this class it could be worth it. Libfmt uses pointers here fwiw